### DISCUSSION TITLE
Double Dirac spinor polarization's properties

### DISCUSSION THREAD

**Discussion by manonsaturn89** [FeynCalc/feyncalc#154] [Q&A]
I am trying to calculate the matrix element 
`(SpinorUBar[kp, m].GA[a].SpinorU[k, m]) (SpinorUBar[k, m].GA[b].SpinorU[kp, m])`
and I was wondering if there is a way to include the polarization degrees of freedom. 

I am interested in the cross section of the initial polarized particles, in this case I would like to make automatic use of the identity
`SpinorU[k, m]SpinorUBar[k, m] = (GS[k]+m)(1+GA[5].GS[s])/2`
where s is the initial particle polatization vector, for the initial particle state, while I still would like to be able to use FermionSpinSum for the final state.

How can I accomplish that?

**Comment by vsht** [comment]
You can do something like

```
ampSq = FCI[(SpinorUBar[kp, m] . GA[a] . 
     SpinorU[k, m]) (SpinorUBar[k, m] . GA[b] . SpinorU[kp, m])]
```

```
FCDiracIsolate[ampSq,Head->dch]/.DOT->holdDOT/.{
dch[holdDOT[a1__, Spinor[Momentum[k], m, __]]]dch[holdDOT[Spinor[Momentum[k], m,__],a2__]]:>dch[holdDOT[a1,1/2( 1+ la GA[5] . GS[S]) . (GS[k]+m),a2]]
}
%/.holdDOT->DOT/.dch->Identity
%//FermionSpinSum//DiracSimplify
```
